chapters/characterization_tests/angle_sensitivity.tex

```diff
@@ -0,0 +1,75 @@
+\section{Sensitivity to Deterministic Angle Variations}
+\label{sec:DetAngle}
+[Describe problem selection decision.] \\
+
+[Describe physical reasons why these are important for testing the method.] \\
+
+\subsection{Parametric Study Description}
+\label{subsec:parstudy}
+
+[Describe selection of deterministic parameters to test angle sensitivity] \\
+
+[Emphasize why other deterministic parameters are omitted.] \\
+
+\subsection{Quadrature Type}
+\label{subsec:quadtype}
+
+[Show FOMs as a function of quad type for first problem] \\
+
+[Show interesting anisotropy plots for extreme-valued quad types.] \\
+
+[Show flux maps for each extreme-valued problem in region of interest.] \\
+
+[Describe results.] \\
+
+[Show FOMs as a function of quad type for second problem] \\
+
+[Show interesting anisotropy plots for extreme-valued quad types.] \\
+
+[Show flux maps for each extreme-valued problem in region of interest.] \\
+
+[Describe results.] \\
+
+\subsection{Quadrature Order}
+\label{subsec:quadorder}
+[Show FOMs as a function of quad order for first problem] \\
+
+[Show interesting anisotropy plots for extreme-valued quad types.] \\
+
```

Comment:
Good point. My thoughts on this are maybe that there should be a sister appendix to this chapter that includes all of the results from each problem that aren't included in this chapter. How do you feel about that?